source/docs/software/wpilib-tools/robotbuilder/introduction/starting-robotbuilder.rst

```diff
@@ -0,0 +1,25 @@
+Starting RobotBuilder
+=====================
+
+.. note:: RobotBuilder is a Java program and as such should be able to run on any platform that is supported by Java. We have been running RobotBuilder on Mac OS X, Windows, and various versions of Linux successfully.
+
+Getting RobotBuilder
+--------------------
+
+RobotBuilder is downloaded as part of the WPILib Offline Installer. For more information, see the :ref:`Windows/MacOS/Linux installation guides <docs/software/getting-started/index:Getting Started>`
+
+Option 1 - Starting from VSCode
+-------------------------------
+
+.. image:: images/starting-robotbuilder-1.png
+
+Press **Ctrl + Shift + P** and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette and select **Start Tool**, then select **Robot Builder**.
+
+Option 2 - Running from the Script
+----------------------------------
+
+.. image:: images/starting-robotbuilder-2.png
+
+The install process installs the tools to ``~home\frcYYYY\tools`` (where YYYY is the year and ~home is ``C:\Users\Public on Windows``).
```

Comment:
```suggestion
The install process installs the tools to ``~/frcYYYY/tools`` (where YYYY is the year and ~ is ``C:\Users\Public on Windows``).
```

Reply:
Any particular reason for this change?

Reply:
Because `~` means user home, so ~home\ is wrong. And for linux, the path seperator is `/`

Reply:
Ok